src/js/components/Button/StyledButtonKind.js

```diff
@@ -37,7 +37,14 @@ const fontStyle = (props) => {
   `;
 };
 
-const padFromTheme = (size = 'medium', theme, themeObj) => {
+const padFromTheme = (size = 'medium', theme, themeObj, kind) => {
+  if (size && themeObj?.size?.[size]?.[kind]) {
```

Comment:
I think this might crash if we have `themeObj?.size?.[size]?.[kind]` but not `themeObj?.size?.[size]?.[kind].pad`

Reply:
Correct, fixed.